### PR TITLE
bugfix/preparation-kit

### PR DESCRIPTION
**Pull request by PCOffline**

### CODE REVIEW

**Comment by PCOffline** [comment]
Compatible with https://github.com/alphayesodot/itur-be-functionapp/pull/128

File: src/pages/PreparationKit/index.jsx [hunk 1]
```diff
@@ -49,20 +52,17 @@ const PreparationKit = ({ sidebarRef }) => {
...
+    <Document file={document} loading={<Typography>{t('text.loading')}</Typography>} onLoadSuccess={onDocumentLoadSuccess}>
       <Page
         pageNumber={pageNumber}
         className={classes.document}
         height={isFullscreen ? undefined : pageHeight - pageInfoRef.current?.clientHeight}
       />
     </Document>
-  );
+    ) : (
+      <p>{documentMessage}</p>
```
Comment (carmeltsadok) [inline src/pages/PreparationKit/index.jsx:64]: use `NoObjectsToShow` component

File: src/pages/PreparationKit/index.jsx [hunk 2]
```diff
@@ -108,7 +102,7 @@ const PreparationKit = ({ sidebarRef }) => {
           {renderDocument()}
         </div>
         <div className={classes.playerWrapper}>
-          <ReactPlayer url={video} className={classes.player} controls />
+          {video ? <ReactPlayer url={video} className={classes.player} controls /> : <p>{videoMessage}</p>}
```
Comment (carmeltsadok) [inline src/pages/PreparationKit/index.jsx:105]: same

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: mockserver/preparationKit/preparationKit.manager.js [hunk 3]
```diff
@@ -4,7 +4,24 @@ import { fileURLToPath } from 'url';
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
 export default class PreparationKitManager {
-  static getFile(req, res) {
+  static getBlob(req, res) {
     res.sendFile(`${__dirname}/files/${req.params.fileName}`);
   }
+
+  static getSas(req, res) {
```
Comment (lustigy96) [inline mockserver/preparationKit/preparationKit.manager.js:11]: who is Sas? I want to get it too

Comment (lustigy96) [reply mockserver/preparationKit/preparationKit.manager.js:11]: write a comment to explain what Sas is and explain it's use

File: mockserver/preparationKit/preparationKit.manager.js [hunk 4]
```diff
@@ -4,7 +4,24 @@ import { fileURLToPath } from 'url';
...
+    switch (req.params.fileType) {
+      case 'video':
+      case 'mp4':
+        res.send('https://sample-videos.com/video123/mp4/720/big_buck_bunny_720p_1mb.mp4');
```
Comment (lustigy96) [inline mockserver/preparationKit/preparationKit.manager.js:15]: put this url in const under the name mp4_example

File: mockserver/preparationKit/preparationKit.manager.js [hunk 5]
```diff
@@ -4,7 +4,24 @@ import { fileURLToPath } from 'url';
...
+        break;
+
+      case 'document':
+      case 'pdf':
+        res.send('https://file-examples-com.github.io/uploads/2017/10/file-sample_150kB.pdf');
```
Comment (lustigy96) [inline mockserver/preparationKit/preparationKit.manager.js:20]: put this url in const under the name pdf_example

File: src/services/preparationKit.service.js [hunk 6]
```diff
@@ -2,13 +2,19 @@ import axios from 'axios';
...
+      url: `${config.apiUri}/api/preparation-kit/blob/${fileName}`,
       responseType: 'blob',
       reponseEncoding: 'binary',
     });
     return data;
   }
+
+  static async getSasByType(fileType) {
+    const { data } = await axios.get(`${config.apiUri}/api/preparation-kit/sas/${fileType}`);
+
```
Comment (lustigy96) [inline src/services/preparationKit.service.js:17]:
```suggestion
```

**Review by lustigy96 — CHANGES_REQUESTED** [review]

**Review by Delitefully — APPROVED** [review]

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**